src/main/scala/org/opennetworkinsight/proxy/ProxySchema.scala

```diff
@@ -0,0 +1,39 @@
+package org.opennetworkinsight.proxy
+
+object ProxySchema {
+
+  // fields from the input
+
+  val Date = "p_date"
+  val Time = "p_time"
+  val ClientIP = "clientip"
+  val Host = "host"
+  val ReqMethod = "reqmethod"
+  val UserAgent = "useragent"
+  val ResponseContentType = "resconttype"
```

Comment:
i should look up what the actual full field names are for these things... I just extrapolated this one